lib/index.js

```diff
@@ -273,6 +273,48 @@ exports.createStorage = async (ctx) => {
     }
 };
 
+exports.deleteStorage = async (ctx) => {
+    debug("DELETE STORAGE");
+
+    // Find the Client
+    const issuerId = `uri:fusebit-template:${ctx.functionId}:${ctx.body.subscriptionId}:${ctx.body.boundaryId}:${ctx.body.functionId}`;
+    const subject = 'client-1';
```

Comment:
I'd lift the creation of (issuer, subject) to a separate function instead of replicating this in createStorage and deleteStorage.